src/server/test/web/readingsBarGroupQuantity.js

```diff
@@ -48,8 +48,24 @@ mocha.describe('readings API', () => {
                             graphicUnitId: unitId });
                     // Check that the API reading is equal to what it is expected to equal
                     expectReadingToEqualExpected(res, expected, GROUP_ID);
-            });
+                });
                 // Add BG4 here
```

Comment:
Could you please remove the comment about adding the test since done. The one for BG3 was accidently left so it was confusing. If you are willing to remove that too it would be nice.